File: pubsub-springboot-std/src/main/java/com/example/std/gettingstarted/Application.java
package com.example.std.gettingstarted;


import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import springfox.documentation.swagger2.annotations.EnableSwagger2;


@EnableSwagger2
@SpringBootApplication
@ComponentScan(basePackages = {"com.example.std.gettingstarted.config",
                              "com.example.std.gettingstarted.controllers"
                                })
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }




}

File: appengine-standard-java8/kotlin-springboot-appengine-standard/src/main/kotlin/Application.kt
package org.jetbrains.kotlin.demo

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.web.support.SpringBootServletInitializer

@SpringBootApplication
open class Application : SpringBootServletInitializer(){
    companion object {
        @JvmStatic fun main(args: Array<String>) {
            SpringApplication.run(Application::class.java, *args)
        }
    }
}
